Add some extra video modes and change the default to VGA.

---
 hw/kdrive/src/kmode.c |   41 +++++++++++++++++++++++++++++++++++++++--
 1 file changed, 39 insertions(+), 2 deletions(-)

Index: xorg-server-1.3.0.0/hw/kdrive/src/kmode.c
===================================================================
--- xorg-server-1.3.0.0.orig/hw/kdrive/src/kmode.c	2008-01-11 14:20:47.000000000 +0000
+++ xorg-server-1.3.0.0/hw/kdrive/src/kmode.c	2008-01-11 14:21:18.000000000 +0000
@@ -32,6 +32,31 @@ const KdMonitorTiming  kdMonitorTimings[
     /*	H	V	Hz	KHz */
 		/*  FP	    BP	    BLANK   POLARITY */
 
+	/* Treo 650 */
+
+    {   320,	320,	64,	16256,
+	            17,	    12,	    32,     KdSyncNegative,
+	            1,      11,     14,     KdSyncNegative,
+    },
+    
+    {   320,	320,	64,	    0,
+	            0,	    0,	    0,     KdSyncNegative,
+	            0,      0,      0,     KdSyncNegative,
+    },
+	
+	/* LifeDrive/T3/TX modes */
+
+    {   320,	480,	64,	16256,
+	            17,	    12,	    32,     KdSyncNegative,
+	            1,      11,     14,     KdSyncNegative,
+    },
+    
+    {   480,	320,	64,	    0,
+	            0,	    0,	    0,     KdSyncNegative,
+	            0,      0,      0,     KdSyncNegative,
+    },
+
+
     /* IPAQ modeline:
      *
      * Modeline "320x240"      5.7222 320 337 340 352   240 241 244 254"
@@ -41,6 +66,11 @@ const KdMonitorTiming  kdMonitorTimings[
 	            1,      11,     14,     KdSyncNegative,
     },
     
+    {   240,	320,	64,	    0,
+	            0,	    0,	    0,     KdSyncNegative,
+	            0,      0,      0,     KdSyncNegative,
+    },
+
     /* Other VESA modes */
     {	640,	350,	85,	31500,			    /* VESA */
 		    32,	    96,	    192,    KdSyncPositive, /* 26.413 */
@@ -76,10 +106,19 @@ const KdMonitorTiming  kdMonitorTimings[
 		    16,	    120,    176,    KdSyncNegative, /* 37.861 */
 		    1,	    20,	    24,	    KdSyncNegative, /* 72.809 */
     },
+    /* DEFAULT */
+#define MONITOR_TIMING_DEFAULT	13
     {	640,	480,	60,	25175,			    /* VESA */
 		   16,	    48,	    160,    KdSyncNegative, /* 31.469 */
 		   10,	    33,	    45,	    KdSyncNegative, /* 59.940 */
     },
+
+
+  {   480,    640,    60,     0,                      /* VESA */
+               0,      0,     0,    KdSyncNegative, /* 31.469 */
+               0,      0,     0,     KdSyncNegative, /* 59.940 */
+   },
+  
     
     /* 800x600 modes */
     {	800,	600,	85,	56250,			    /* VESA */
@@ -90,8 +129,6 @@ const KdMonitorTiming  kdMonitorTimings[
 		    16,	    160,    256,    KdSyncPositive, /* 46.875 */
 		    1,	    21,	    25,	    KdSyncPositive, /* 75.000 */
     },
-    /* DEFAULT */
-#define MONITOR_TIMING_DEFAULT	9
     {	800,	600,	72,	50000,			    /* VESA */
 		    56,	    64,	    240,    KdSyncPositive, /* 48.077 */
 		    37,	    23,	    66,	    KdSyncPositive, /* 72.188 */
